src/components/menu/Menu.md

```diff
@@ -7,14 +7,13 @@ The component handles adding aria attributes, keyboard navigation, and focus.
 The `<MenuLinkItem>` component is required to wrap anchor tags since there are special ARIA rules for those.
 The `<SelectMenuLinkItem>` component is for items that can be selected with a checkmark.
```

Comment:
Why are these docs and code snippet duplicated across `Menu.md` and `SelectMenuLinkItem.md`? Would it be better to show at least one example of `MenuLinkItem`?